Scripts/Levels/Tutorial/1 - Game Intro.gd

```diff
@@ -32,7 +34,11 @@ var robotStartOrientation = PEP.Orientation.RIGHT
 
 func _ready():
 	PEP.loadLevel(self)
-	DialogueManager.add_dialogue(self, "Tutorial/1 - Game Intro.txt")
+	var root = get_tree().root
+	var levelPath = root.get_child(root.get_child_count() - 1).filename
+	if GameStats.playTutorial[levelPath]:
+		DialogueManager.add_dialogue(self, textPath)
+		GameStats.playTutorial[levelPath] = false
```

Comment:
Could we potentially have this encapsulated in the DialogueManager add_dialogue code so we don't have every level script's _ready() func get cluttered?